python/lesson3/tutorial.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Discovering Flask via Python and pip
+---
+
+An alternative to *Ruby's Sinatra* is the lesser known *Flask* (run using Python).
+Both are web application frameworks. Similar to the MVC (model-view-controller)
```

Comment:
Also, these guys probably don't know (or care, yet) what MVC is. Diving into that first is going to be tricky.